archipelago-client/ArchipelagoInterface.cpp

```diff
@@ -67,14 +73,21 @@ BOOL CArchipelago::Initialise(std::string URI) {
 			ap->ConnectUpdate(false, 1, true, tags);
 		}
 
+		Core->connected = true;
+		ItemRandomiser->sendMissedItems();
+
 		});
 	ap->set_slot_disconnected_handler([]() {
-		Core->Logger("Slot disconnected");
+		spdlog::info("Slot disconnected");
+		Core->connected = false;
+		GameHook->showBanner(L"Archipelago disconnected! Don't pick up any items until it reconnects");
```

Comment:
Maybe this behavior belongs also to set_socket_disconnected_handler, I need to give it a try